### PR TITLE
[feat/website] : temporary website 

### PR DESCRIPTION
**Pull request by Qouagga**
Temporary website waiting for new version of the website

### CODE REVIEW

File: website/docs/user_guide_annotator.md [hunk 1]
```diff
@@ -1,114 +0,0 @@
-# APLOSE Annotator User Guide
```
Comment (ixio) [inline website/docs/user_guide_annotator.md:1]: Why delete all of this and the related images ?

File: website/src/components/CardArticle/index.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  img, // images must be squares
+  imgAlt = '',
+  stringDate = "",
+  title,
+  article,
+  authors,
+  children
+}) => {
+  const regex = /\//g;
+  const d_m_years = stringDate.split(regex);
```
Comment (ixio) [inline website/src/components/CardArticle/index.tsx:25]: Why not just `"2022/03/01".split("/")`  ?

> Some people, when confronted with a problem, think "I know, I'll use regular expressions." Now they have two problems.

You can read a bit more context of the quote here : https://blog.codinghorror.com/regular-expressions-now-you-have-two-problems/ :grin:.
I'd say don't use regex when you can do simpler code, only use regex when really needed.

File: website/src/components/Layout/Header/styles.css [hunk 3]
```diff
@@ -32,4 +35,10 @@ text-align: center;
   nav li.pipe{
     display: inline-block !important;
   }
+}
+
+.dropdown-menu{
+  background-color: #f8f9fa !important;
+  padding: 0.5rem 4.5em !important;
+  margin-left: -4em;
 }
```
Comment (ixio) [inline website/src/components/Layout/Header/styles.css:44]: The github PR shows you're missing a newline at the end of your file. You have that in other files. Always add an empty newline at file ends (it helps with bash tools).

**Review by ixio — COMMENTED** [review]
Overall looks good, just a few comments. Let me know if this has been tester by the team and approved.

File: website/docs/user_guide_annotator.md (hunk 1 above)
Comment (Qouagga) [reply website/docs/user_guide_annotator.md:1]: C'était suite à ta demande, les docs sont tous regroupés dans le répertoire docs d'osmose-app

Comment (ixio) [reply website/docs/user_guide_annotator.md:1]: Oh right c'était une copie que tu as supprimé.

**Review by flo-bou — COMMENTED** [review]
Ces derniers changements ont l'air bons.
Petites remarques :
* la photo de Mathieu est déformée. L'image devrait être un carré pour avoir un bon rendu
* le titre sur la page 'Publication' est peu lisible
* plusieurs petits problèmes d'affichage sur la page news

File: website/src/components/PageTitle/index.tsx [hunk 4]
```diff
@@ -7,23 +7,24 @@ export interface PageTitleProps {
   img?: string;
   imgAlt?: string;
   imgSet?: string;
+  parallax?: boolean;
```
Comment (rvovard) [inline website/src/components/PageTitle/index.tsx:10]: Is this parameter used?

**Review by rvovard — CHANGES_REQUESTED** [review]
A few comments, but a great work otherwise, especially the tutorial!

**Review by ixio — APPROVED** [review]
Like I said in Slack, re-test this PR and if it works merge it as is. If there is still stuff I don't like I'll suggest a refactor later on. If there is stuff that is wrong let it break and make a bugfix quickly. Godspeed.

**Comment by flo-bou** [comment]
Il y a 2 fois la section sur Anatole dans la page 'People' (il y a du favoritisme !), il faudrait retirer la première.
La nouvelle image dans la page "Publications" est assez lourde (3.3 Mo), elle ralentit le chargement. Mais cela pourra être corrigé plus tard si tu n'as pas le temps.
Le reste me parait tout bon :-)

**Review by flo-bou — APPROVED** [review]

**Pull request merged**